ocaml/xapi/cli_operations.ml

```diff
@@ -32,6 +32,8 @@ open Records
 let failwith str = raise (Cli_util.Cli_failure str)
 exception ExitWithError of int
 
+let (+++) f g a b = f (g a b)
```

Comment:
Could we make the code that uses `+++` more readable?

Alternatively I would rather get `+++` defined in the `vm_migrate` scope only, where it's used